Sudoku.Application/CreateSudoku.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace Sudoku.Application
+{
+    public class CreateSudoku
+    {
+        public CreateSudoku(int size, int complexity)
+        {
+        }
+
+        public void DeleteCells()
```

Comment:
how many cells will be deleted. Should we specify this number as a parameter( for example defined by complexity)